packages/client/pages/cover/print.vue

```diff
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+import { watchEffect } from 'vue'
+import { useNav } from '../../composables/useNav'
+import { themeVars } from '../../env'
+import HandoutCover from '#slidev/global-components/handout-cover'
+
+const { isPrintMode } = useNav()
+
+watchEffect(() => {
+  const html = document.body.parentNode as HTMLElement
```

Comment:
I think maybe we don't need a `.print` class here. Because when this component is loaded, the page must be in the print mode.